src/CiviliansAccountsMicroservice/Civilians.Application/ViewModels/Passports/UpdatePassportViewModel.cs

```diff
@@ -9,8 +9,6 @@ namespace Civilians.Application.ViewModels.Passports
 
     public class UpdatePassportViewModel
     {
-        [Required]
-        public Guid Id { get; set; } = Guid.Empty;
         [Required]
```

Comment:
What do you think about fluent validation?

Reply:
I think it's better to complete obligatory tasks first, then to do other less important things like removing empty lines, adding ones, moving braces, etc.
In the Issues tab I've added an issues for refactoring with adding fluent validation.

And about fluent validation, I think it's unnecessary to create separate class for just a Required tag.